frontend/src/utils/GeneralUtils.ts

```diff
@@ -94,3 +96,11 @@ export const getLetterFromNumber = (number: number): string => {
 
   return firstLetter + secondLetter;
 };
+
+export const getQueryName = (query: DocumentNode): string => {
+  const name = getOperationAST(query)?.name?.value;
+  if (!name) {
+    throw new Error("Query name not found. This is probably a bug.");
```

Comment:
is there some info about the query we can add into the error message? or the query itself lol

Reply:
good point lol